request_a_govuk_domain/request/utils.py

```diff
@@ -38,14 +39,17 @@ def is_central_government(registrant_type: str) -> bool:
     return registrant_type in ["central_government", "ndpb"]
 
 
-def add_to_session(form, request, field_name: str) -> str:
+def add_to_session(form, request, field_names: List[str]) -> str:
     """
-    Common utility method to clean the field and save it in the session
-    Returns the field value
-    This is to save boilerplate code in the views
+    Common utility method to clean the list of fields and save them in the session. This is to save boilerplate code.
+    Returns:
+        cleaned field value
+        registration data
+    Either/Both the return values are used in the views
     """
```

Comment:
Could you use the `:param:` format (as in domains-api) to formally describe each parameter and return value? We don't format it in formal documentation yet, but we might one day